<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMDI</class>
 <widget class="QMainWindow" name="TestMDI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1550</width>
    <height>883</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1550</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TestMDI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1521</width>
      <height>781</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1550</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actDoc_New"/>
   <addaction name="actDoc_Open"/>
   <addaction name="actViewMode"/>
   <addaction name="actCascade"/>
   <addaction name="actTile"/>
   <addaction name="actCloseALL"/>
   <addaction name="actFont"/>
   <addaction name="actCopy"/>
   <addaction name="actCut"/>
   <addaction name="actPaste"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actDoc_New">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/100.bmp</normaloff>:/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>新建文档</string>
   </property>
   <property name="toolTip">
    <string>新建文档窗口</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/132.bmp</normaloff>:/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本系统</string>
   </property>
  </action>
  <action name="actDoc_Open">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/122.bmp</normaloff>:/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文档</string>
   </property>
   <property name="toolTip">
    <string>打开文档</string>
   </property>
  </action>
  <action name="actFont">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/506.bmp</normaloff>:/images/506.bmp</iconset>
   </property>
   <property name="text">
    <string>字体设置</string>
   </property>
   <property name="toolTip">
    <string>字体设置</string>
   </property>
  </action>
  <action name="actCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/200.bmp</normaloff>:/images/200.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/202.bmp</normaloff>:/images/202.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/204.bmp</normaloff>:/images/204.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actViewMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/230.bmp</normaloff>:/images/230.bmp</iconset>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
   <property name="toolTip">
    <string>窗口模式或页面模式</string>
   </property>
  </action>
  <action name="actCascade">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/400.bmp</normaloff>:/images/400.bmp</iconset>
   </property>
   <property name="text">
    <string>级联展开</string>
   </property>
   <property name="toolTip">
    <string>窗口级联展开</string>
   </property>
  </action>
  <action name="actTile">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/406.bmp</normaloff>:/images/406.bmp</iconset>
   </property>
   <property name="text">
    <string>平铺展开</string>
   </property>
   <property name="toolTip">
    <string>窗口平铺展开</string>
   </property>
  </action>
  <action name="actCloseALL">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/128.bmp</normaloff>:/images/128.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭全部</string>
   </property>
   <property name="toolTip">
    <string>关闭所有窗口</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>TestMDI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
